configs/faster_rcnn/faster-rcnn_r50-caffe-dc5_ms-3x_coco.py

```diff
@@ -1,4 +1,4 @@
 _base_ = './faster-rcnn_r50-caffe-dc5_ms-1x_coco.py'
-# learning policy
-lr_config = dict(step=[28, 34])
-runner = dict(type='EpochBasedRunner', max_epochs=36)
+
+_base_.param_scheduler[1].milestones = [28, 34]
+train_cfg = dict(max_epochs=36)
```

Comment:
Why the above one uses setting base value, while the below uses a dict-like cfg?

Reply:
The main thing is which one is convenient to use which one. Providing a diverse range of writing methods is also convenient for reference

Reply:
then let's add some comments to explain it. for example, tell users there are another way to do that in comments.

Reply:
OK